docs/troubleshooting.md

```diff
@@ -44,3 +44,26 @@ Result:
 ```sh
 {"value":1.23,"reason":"DEFAULT","variant":"one"}
 ```
+
+---
+
+## Extra/Duplicate Events in File Syncs
+
+When using the file sync, you may notice more events than you'd expect.
+This is attributable to nuances in text editors and the OS/filesystem.
+Most editors will cause a few filesystem events on a save, for instance, not just one as you might expect.
```

Comment:
```suggestion
Most editors will cause a few file system events on a save, for instance, not just one as you might expect.
```

Reply:
I think this is awkward. If always written this as one word (and most Linux foundation documentation I've read uses "filesystem")

https://www.google.com/amp/s/www.linuxfoundation.org/blog/blog/classic-sysadmin-the-linux-filesystem-explained%3fhs_amp=true

Reply:
If it's good enough for a random Linux blog it's good enough for me 😀.